bundles/org.openhab.binding.energidataservice/src/main/java/org/openhab/binding/energidataservice/internal/action/EnergiDataServiceActions.java

```diff
@@ -100,7 +101,7 @@ public class EnergiDataServiceActions implements ThingActions {
     }
 
     @RuleAction(label = "@text/action.calculate-price.label", description = "@text/action.calculate-price.description")
-    public @ActionOutput(name = "price", type = "java.math.BigDecimal") @Nullable BigDecimal calculatePrice(
+    public @ActionOutput(label = "Price", type = "java.math.BigDecimal") @Nullable BigDecimal calculatePrice(
```

Comment:
Is I18N supported for labels?

Reply:
In that case, can you add I18N texts for the new labels? Everything else supports I18N, so otherwise there will be mixed languages.

Reply:
Makes sense.

Reply:
Done